metecho/api/views.py

```diff
@@ -194,6 +217,13 @@ def get_queryset(self):
         repo_ids = self.request.user.repositories.values_list("repo_id", flat=True)
         return self.queryset.filter(repo_id__in=repo_ids)
 
+    def create(self, request):
```

Comment:
We need to make sure this is only available to users that have been granted a particular Django permission.

Reply:
Should we also require they be a member of the target organization? Or just the Django permission?

Reply:
I don't think we can add them to a team unless they are a member

Reply:
That's what I thought, but apparently the [GH API endpoint](https://docs.github.com/en/rest/reference/teams#add-or-update-team-membership-for-a-user) won't check for us and will send an invite if the user is outside the org.

> An organization owner can add someone who is not part of the team's organization to a team. When an organization owner adds someone to a team who is not an organization member, this endpoint will send an invitation to the person via email. This newly-created membership will be in the "pending" state until the person accepts the invitation, at which point the membership will transition to the "active" state and the user will be added as a member of the team.

Reply:
@jerivas I don't think we want Metecho to be able to issue that invite because only Salesforce staff should be adding people to the organization.

Reply:
Fixed in f7fa08fff1356ede3e7f3da991d1804d7a5fcae4